Dockerfile.DOVECOT

```diff
@@ -1,31 +0,0 @@
-FROM fedora:24
```

Comment:
Is this image going to be moved to separate github repo?

Reply:
@TomasTomecek Yes. https://github.com/container-images/dovecot. But I will do on it later on.

Reply:
awesome!